.github/workflows/rust.yml

```diff
@@ -37,43 +41,44 @@ jobs:
           toolchain: ${{ matrix.rust }}
 
       - name: Build
-        run: cargo +${{ matrix.rust }} build
+        run: cargo +${{ matrix.rust }} build --features=${{ matrix.features }}
```

Comment:
nit: a bit of harmless redundancy here - there are no features in this package :sweat_smile:

Reply:
done as well